src/client/state/RoomsInput.js

```diff
@@ -401,7 +404,7 @@ class RoomsInput extends EventEmitter {
     };
 
     // Apply formatting if relevant
-    const formattedBody = this.formatAndEmojifyText(editedBody);
+    const formattedBody = formatAndEmojifyText(editedBody);
```

Comment:
the first parameter for `formatAndEmojifyText` is `room` which is not passed here causing error while editing.

Reply:
aa that's an embarrassing mistake :p
sorry for forgetting to check editing